nuxt.config.js

```diff
@@ -25,7 +25,18 @@ export default {
 						}
 					})
 
-					callback(null, pageRoutes.concat(postRoutes))
+					const totalPages = posts.headers["x-wp-totalpages"]
```

Comment:
You're only getting a page count here, @TeddTech — you aren't fetching all the pages of posts. On line 11, `axios.get(`${Config.wpDomain}${Config.apiBase}posts`)` will just fetch the first page and then stop. I expect this might be part of the issue.

Reply:
(I know there's only one page of posts to fetch right now, but it's going to break for sure when there are more.)

Reply:
I believe it's fixed now

Reply:
@TeddTech What happens if we have more than 100 posts?